src/components/playback/playbackmanager.js

```diff
@@ -57,7 +63,14 @@ define(['events', 'datetime', 'appSettings', 'itemHelper', 'pluginManager', 'pla
             addPlaylistToPlaybackReport(playbackManagerInstance, info, player, serverId);
         }
 
-        console.debug(method + '-' + JSON.stringify(info));
+        const now = (new Date).getTime();
+
+        if (method !== reportPlaybackLastMethod || now - (reportPlaybackLastTime || 0) >= 1e3) {
+            console.debug(method + '-' + JSON.stringify(info));
+            reportPlaybackLastMethod = method;
+            reportPlaybackLastTime = now;
+        }
```

Comment:
don't we need an `else return` clause here? otherwise we're not spamming the console but _are_ reporting the playback

Reply:
All "magic" in `ApiClient.reportPlaybackProgress` - it remembers last update options to send most recent one.
`ApiClient.reportPlaybackStart` and `ApiClient.reportPlaybackStopped` should be invoked without limits.